MIGRATION.md

```diff
@@ -89,6 +89,15 @@ The **English**, **Spanish**, **German**, and **French** copy for the following
 - `auth_retry.subtitle`
 - `auth_retry.title`
 - `auth_upload_pass.body`
+- `welcome.doc_video_subtitle`
+- `welcome.list_header_doc_video`
+- `welcome.list_item_doc`
+- `welcome.list_item_doc_video_timeout`
+- `welcome.list_item_selfie`
+
```

Comment:
These should go under the next version migration. Also make sure that all the new strings are added too

Reply:
All updated :point_up: